modules/apps/27-interchain-accounts/types/validate.go

```diff
@@ -0,0 +1,41 @@
+package types
+
+import (
+	"regexp"
+	"strings"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// DefaultMaxAddrLength defines the default maximum character length used in validation of addresses
+var DefaultMaxAddrLength = 128
+
+// IsValidAddr defines a regular expression to check if the provided string consists of
+// strictly alphanumeric characters
+var IsValidAddr = regexp.MustCompile("^[a-zA-Z0-9]*$").MatchString
+
+// ValidateVersion performs basic validation of the provided ics27 version string
+// When no delimiter is present it compares against the expected version
```

Comment:
I added this comment for line 21 I think rather than the header.

Reply:
I think it's probably more appropriate to include it as part of the godoc rather than inline. [See since updated](https://github.com/cosmos/ibc-go/pull/410/files#diff-094fd8ce1e0b97d12d88f0711bf8bf51c1ad6022dd68b95db8b3b9830bfad99dR17)

Reply:
great point!